src/Helpers.elm

```diff
@@ -18,28 +20,12 @@ orLazy ma fmb =
     Just _ -> ma
 
 
-{-| Convert a list of `Maybe a` to a list of `a` only for the values different from `Nothing`.
-
-    values [ Just 1, Nothing, Just 2 ] == [ 1, 2 ]
-
--}
-values : List (Maybe a) -> List a
-values = List.foldr foldrValues []
-
-
-foldrValues : Maybe a -> List a -> List a
-foldrValues item list =
-  case item of
-    Nothing -> list
-    Just v -> v :: list
-
-
-
--- List ------------------------------------------------------------------------
-
+segments : Bool -> List ( Float, Float ) -> List ( ( Float, Float ), ( Float, Float ) )
+segments closed ps =
+    List.map2 Tuple.pair ps (rotate ps)
+        |> (if closed then
+                identity
```

Comment:
Oh yeah, sounds good.

I like the separate `Helpers.List` because as you mentioned before it allows for `Helpers.List.rotate` instead of `rotateList`.  The mostly empty `Helpers` module also doesn't bother me - it just provides a place for future helper functions.